cypress/support/commands.js

```diff
@@ -82,3 +82,15 @@ Cypress.Commands.add('typeCmdOrCtrl', () => {
 function isMac() {
     return navigator.platform.toUpperCase().indexOf('MAC') >= 0;
 }
+
+// ***********************************************************
+// Post
+// ***********************************************************
+
+Cypress.Commands.add('post', (message) => {
+    cy.get('#post_textbox').type(message).type('{enter}');
+});
+
+Cypress.Commands.add('getLastPostId', () => {
+    return cy.get('#postListContent').children().last().invoke('attr', 'id');
```

Comment:
Please return actual post ID, maybe something like:
```suggestion
    return cy.get('#postListContent').children().last().invoke('attr', 'id').then((divPostId) => {
        return divPostId.replace('post_', '');
    });
```